lua/obsidian/filewatch.lua

```diff
@@ -109,7 +109,7 @@ local function watch_path(path, on_event, on_error, opts)
 
     local folder_path = uv.fs_event_getpath(handle)
 
-    local full_path = table.concat { folder_path, filename }
+    local full_path = table.concat { folder_path, "/", filename }
```

Comment:
```suggestion
    local full_path = vim.fs.joinpath(folder_path, filename)
```
this makes sure path separator is correct on windows.